target/linux/ramips/mt7620/profiles/zbt.mk

```diff
@@ -6,11 +6,11 @@
 #
 
 define Profile/ZBT-WE826
-	NAME:=ZBT-WE826 Device
+	NAME:=ZBT-WE826
 	PACKAGES:=\
-		kmod-scsi-core kmod-scsi-generic kmod-mmc kmod-sdhci \
-		block-mount kmod-usb-core kmod-usb-dwc2 kmod-usb2 kmod-usb-ohci \
-		kmod-mt76
+		kmod-mt76 kmod-scsi-core kmod-scsi-generic kmod-mmc kmod-sdhci \
+		block-mount kmod-usb-core kmod-usb2 kmod-usb-ohci kmod-usb-serial \
+		kmod-usb-serial-option kmod-usb-serial-wwan comgt wwan
```

Comment:
The general convention is to deploy image with only basic and really needed packages.

This model is available (directly from manufacturer) without the WWAN card, so you shouldn't add custom packages here as the user will be able to install them later.

Remember that there is no way to remove packages which are included/builtin in squashfs image - removing/uninstalling them will not free up space on the FLASH.